core_weekly/core_weekly.py

```diff
@@ -28,8 +28,8 @@ def __init__(self, args):
         self.date_range = args.date
         if args.week is not None:
             self.date_range = DateUtil().get_date_range_from_week(args.week, args.year)
+            self.week = args.week
 
-        if self.date_range:
```

Comment:
`self.date_range` is always defined if I am not wrong (line 28)

Reply:
It can be null or empty, this check is not a defined state

Reply:
So I revert my change https://github.com/PrestaShop/core-weekly-generator/pull/90/commits/6175d78ea8fdbd060c042523cd8a2abbfa98b83b